src/proto/api_messages_test.js

```diff
@@ -15,17 +15,19 @@
  */
 
 import {
-    AnalyticsContext,
-    AnalyticsRequest,
-    AnalyticsEvent,
-    deserialize} from './api_messages';
+  AnalyticsContext,
+  AnalyticsEventMeta,
+  AnalyticsRequest,
+  AnalyticsEvent,
+  EventOriginator,
+  deserialize} from './api_messages';
 
 /**
- * Compare two protos
- * @param {!Array} thisArray
- * @param {!Array} otherArray
- * @return {boolean}
- */
+* Compare two protos
```

Comment:
unnecessary indent

Reply:
I added a space before the * so it is more standard.  I'm not sure why I have to manually format this file when I copy it into visual studio.